example/authorizer.yaml

```diff
@@ -1,36 +1,62 @@
-global:
-  REALM: example.com
+default:
+  realm: example.com
 
-  # Options are Bearer, Basic, none
-  WWW_AUTHENTICATE: Basic
-  LOGLEVEL: DEBUG
+  # Generate this with cryptography.fernet.Fernet.generate_key().
+  session_secret: "<session-encryption-secret>"
 
-  # Authorizer Configurations
-  # NO_AUTHORIZE: True
+  # Sample values for scopes.  You can replace these with anything you
+  # want to use for a scope.  Used to populate the new token creation
+  # page.
+  known_capabilities:
+    "admin": Administrative access
+    "read:all": Can read anything
 
-  # group_membership options
-  GROUP_MAPPING:
-    read:workspace: lsst_int_ws_r
-    exec:portal: lsst_int_portal_x
-    exec:notebook: lsst_int_notebook_x
-    read:tap: lsst_int_tap_r
-    read:image: lsst_int_image_r
+  # Map scopes to lists of groups that provide that scope.
+  #
+  # To get scopes based on GitHub team membership, replace <org> with the
+  # name of the GitHub organization and <team> with the name of the team
+  # in that organization.  Members of that team will then get the scope
+  # listed on the left.  (Add as many entries as you'd like.)
+  group_mapping:
+    "admin": ["<org>:<team>"]
 
+  # To get these values, go to Settings > Developer Settings for either a
+  # GitHub user or an organization, go into OAuth Apps, and create a new
+  # application.
+  #
+  # Delete if you don't want to use GitHub authentication.
+  github:
+    client_id: "<github-client-id>"
+    client_secret: "<github-client-secret>"
 
-  OAUTH2_JWT_ISS: https://example.com
-  OAUTH2_JWT_KEY_ID: reissuer-key-id
-  OAUTH2_JWT_KEY: |
-    -----BEGIN RSA PRIVATE KEY-----
-    ...
-    ...
-    ...
-    ...
-    -----END RSA PRIVATE KEY-----
+  # These settings must match the corresponding oauth2_proxy settings.
+  oauth2_store_session:
+    ticket_prefix: "oauth2_proxy"
+    redis_url: redis://redis.example.com:6379/0
+    # Replace with the secret shared with oauth2_proxy.  Must be 256 bits
+    # (32 bytes) encoded in URL-safe base64.
+    oauth2_proxy_secret: "<shared-secret>"
 
-  ISSUERS:
-    https://cilogon.org:
-      audience: cilogon:/client_id/...
+  oauth2_jwt:
+    iss: "https://example.com"
+    key_id: reissuer-key-id
+    aud:
+      default: "https://example.com"
+      internal: "https://example.com/internal"
+    key: |
+      -----BEGIN RSA PRIVATE KEY-----
+      ...
+      ...
+      ...
+      ...
+      -----END RSA PRIVATE KEY-----
```

Comment:
I love that you have it indented in here as an example.  I always mess this up so this will undoubetly save me some time. :)